pkg/registry/core/resourcequota/strategy.go

```diff
@@ -73,6 +73,9 @@ func (resourcequotaStrategy) AllowCreateOnUpdate() bool {
 // ValidateUpdate is the default update validation for an end user.
 func (resourcequotaStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList {
 	errorList := validation.ValidateResourceQuota(obj.(*api.ResourceQuota))
+	if len(errorList) != 0 {
+		return errorList
```

Comment:
validation.ValidateResourceQuotaUpdate may return an error that is not present in the errorList, i.e. it is not duplicate, 

returning here means as long as there is any error here, validation.ValidateResourceQuotaUpdate is skipped, which is not desired.

Reply:
Add comments in the comments below.